orca-plugins-test/src/test/kotlin/com/netflix/spinnaker/orca/plugins/test/OrcaPluginsFixture.kt

```diff
@@ -102,19 +88,3 @@ class OrcaPluginsFixture : PluginsTckFixture, OrcaTestService() {
     versionNotSupportedPlugin = buildPlugin("com.netflix.orca.version.not.supported.plugin", ">=2.0.0")
   }
 }
-
-@SpringBootTest(classes = [Main::class])
-@ContextConfiguration(classes = [PluginTestConfiguration::class])
-@TestPropertySource(properties = ["spring.config.location=classpath:orca-plugins-test.yml"])
-abstract class OrcaTestService
-
-@TestConfiguration
-internal class PluginTestConfiguration {
-
-  @Bean
-  @Primary
-  fun queue(clock: Clock?, publisher: EventPublisher?): Queue {
-    return InMemoryQueue(
-      clock!!, Duration.ofMinutes(1), emptyList(), false, publisher!!)
-  }
-}
```

Comment:
It makes sense to me to lift this out of here and use the Orca instance provided by `orca-api-tck`.